Misc/NEWS.d/next/Core and Builtins/2024-06-05-10-05-05.gh-issue-120037.QyZNce.rst

```diff
@@ -0,0 +1 @@
+Fix user site packages directory being added to `sys.path` when site module is enabled in `._pth` file. Patch by Tanzim Husain.
```

Comment:
```suggestion
Fix user site packages directory being added to :data:`sys.path` when site module is enabled in a ``._pth`` file. Patch by Tanzim Husain.
```

This is a ReST file, not Markdown, so the syntax is slightly different. This should work to allow the docs builds to pass.